docs/api-docs/getting-started/rest-api-authentication.mdx

```diff
@@ -92,7 +92,7 @@ Account-level API accounts are designed to work with the GraphQL Account API, wh
 
 ### Creating account-level credentials
 
-To create an account-level API account, visit the [account control panel](https://accounts.bigcommerce.com/tokens) or the store control panel for a member store, at **[Settings > Account-level API accounts](https://login.bigcommerce.com/deep-links/manage/settings-list)**.
+To create an account-level API account visit the [account control panel](https://accounts.bigcommerce.com/tokens), or the store control panel for a member store at **[Settings > Account-level API accounts](https://login.bigcommerce.com/deep-links/manage/settings-list)**.
```

Comment:
let's go full no commas, then. this is a splice because what's after the comma is a fragment.